Hermes/appscale/hermes/stats/stats_app.py

```diff
@@ -58,357 +53,160 @@ class HandlerInfo(object):
+    '/stats/local/processes': local_processes_stats_handler,
+    '/stats/local/proxies': local_proxies_stats_handler,
+  }
+  return [
+    (route, handler.handler_class, handler.init_kwargs)
+    for route, handler in routes.iteritems()
+  ]
 
-    if self._is_master:
-      # And 3 same kinds of cluster stats
-      self._cluster_nodes_stats = ClusterStats(
-        cache_size=CLUSTER_NODES_STATS_CACHE_SIZE,
-        update_interval=UPDATE_CLUSTER_NODES_STATS_INTERVAL)
-      self._cluster_processes_stats = ClusterStats(
-        cache_size=CLUSTER_PROCESSES_STATS_CACHE_SIZE,
-        update_interval=UPDATE_CLUSTER_PROCESSES_STATS_INTERVAL)
-      self._cluster_proxies_stats = ClusterStats(
-        cache_size=CLUSTER_PROXIES_STATS_CACHE_SIZE,
-        update_interval=UPDATE_CLUSTER_PROXIES_STATS_INTERVAL)
 
-      if not verbose_cluster_stats:
-        # To reduce slave-to-master traffic and verbosity of cluster stats
-        # you can select which fields of stats to collect on master
-        self._cluster_nodes_stats.included_field_lists = IncludeLists({
-          'node': ['utc_timestamp', 'cpu', 'memory',
-                   'partitions_dict', 'loadavg'],
-          'node.cpu': ['percent', 'count'],
-          'node.memory': ['available'],
-          'node.partition': ['free', 'used'],
-          'node.loadavg': ['last_5min'],
-        })
-        self._cluster_processes_stats.included_field_lists = IncludeLists({
-          'process': ['monit_name', 'unified_service_name', 'application_id',
-                      'port', 'cpu', 'memory', 'children_stats_sum'],
-          'process.cpu': ['user', 'system', 'percent'],
-          'process.memory': ['resident', 'virtual', 'unique'],
-          'process.children_stats_sum': ['cpu', 'memory'],
-        })
-        self._cluster_proxies_stats.included_field_lists = IncludeLists({
-          'proxy': ['name', 'unified_service_name', 'application_id',
-                    'frontend', 'backend'],
-          'proxy.frontend': ['scur', 'smax', 'rate', 'req_rate', 'req_tot'],
-          'proxy.backend': ['qcur', 'scur', 'hrsp_5xx', 'qtime', 'rtime'],
-        })
+def get_cluster_stats_api_routes(is_master):
+  """ Creates stats sources and API handlers for providing cluster
+  node, processes and proxies stats (on master node only).
+  If this node is slave, it creates stub handlers for cluster stats routes.
 
-    # All routes (handlers will be assigned during configuration)
-    self._routes = {
-      '/stats/local/node/cache': None,
-      '/stats/local/node/current': None,
-      '/stats/local/processes/cache': None,
-      '/stats/local/processes/current': None,
-      '/stats/local/proxies/cache': None,
-      '/stats/local/proxies/current': None,
-      '/stats/cluster/nodes': None,
-      '/stats/cluster/processes': None,
-      '/stats/cluster/proxies': None,
-    }
-    self._publishers = []
+  Args:
+    is_master: a boolean indicating whether this node is master
```

Comment:
Do you mean `A boolean, indicating whether this node is master`? Should I change the first sign of parameter explanation  to upper case everywhere?
I'm giving up.. where can I find our styling agreement?